ocr.pm

```diff
@@ -47,13 +49,12 @@ sub tesseract {
     }
 
     $img->write($imgfn);
-    if (system('tesseract', $imgfn, $txtfn) == 0) {
-        $txtfn .= '.txt';
-        if (open(my $fh, '<:encoding(UTF-8)', $txtfn)) {
-            local $/;
-            $txt = <$fh>;
-            close $fh;
-        }
+    system('tesseract', $imgfn, $txtfn);
+    $txtfn .= '.txt';
+    if (open(my $fh, '<:encoding(UTF-8)', $txtfn)) {
```

Comment:
Yes, that should be improved. My intention was to really use autodie(:all) and hide all "defensive programming" error checks so I will update the handling of open, too. Thanks for catching this.